ITEventsTests/Mocks/Storages Mocks/EventTagsStorageMock.swift

```diff
@@ -0,0 +1,28 @@
+@testable import ITEvents
+import Promises
+
+class EventTagsStorageMock: IEventTagsStorage {
+    var tags: [Tag] = []
+    
+    init(tags: [Tag] = []) {
+        if !tags.isEmpty {
+            self.tags = tags
+            return
+        }
+        self.tags = [
```

Comment:
Looks like this will never happen as in our test we always have non empty tags array defined.
So this is unused code - should be removed.